Jenkinsfile

```diff
@@ -31,3 +42,37 @@ node('linux') {
 			sh 'make test'
 	}
 }
+
+def git_commit = ''
+def go_list = ''
+
+stage 'Go & Git Info'
+node('linux') {
+
+	def gopath = gopath_linux()
+	def workdir = workdir_linux(gopath)
+
+	dir(workdir) {
+		checkout scm
+
+		// HACK: Recommended approach for getting command output is writing to and then reading a file.
+		sh 'mkdir -p tmp'
+		sh 'go list . > tmp/GO_LIST'
```

Comment:
@vdice I don't think we're missing any steps in the approach. I have a point of confusion, though - I'm noticing that the workflow-cli PR (https://github.com/deis/workflow-cli/pull/128) is getting some env vars (like `SDK_GO_REPO`) that weren't listed in your above comment. Where are those coming from?